src/tainting/Taint.ml

```diff
@@ -65,7 +66,7 @@ let rec _show_call_trace show_thing = function
 (* Signatures *)
 (*****************************************************************************)
 
-type arg_pos = string * int [@@deriving show]
+type arg_pos = string * int [@@deriving show, compare]
 type arg = { pos : arg_pos; offset : IL.name list } [@@deriving show]
```

Comment:
Don't you have to also change this
```ocaml
Stdlib.compare pos1 pos2
```
in `compare_args` ?

Reply:
Yup, just wanted to check the change with you before I went and changed all the Stdlib.compares

Reply:
Oh wait, did I not change any of the compares? Sorry, I must have lost a change at some point. I'll update the PR